env0/resource_project.go

```diff
@@ -73,15 +74,19 @@ func resourceProjectDelete(ctx context.Context, d *schema.ResourceData, meta int
 }
 
 func resourceProjectImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	apiClient := meta.(*client.ApiClient)
-
 	id := d.Id()
-	project, err := apiClient.Project(id)
-	if err != nil {
-		return nil, err
+	var getErr diag.Diagnostics
+	_, uuidErr := uuid.Parse(id)
+	if uuidErr == nil {
+		_, getErr = getProjectById(id, meta)
+	} else {
+		var project client.Project
+		project, getErr = getProjectByName(id, meta)
```

Comment:
I'd add some logging here a-la: "parameter does not seem to be a valid env0 project id - trying to import by name xxx".

Reply:
https://github.com/env0/terraform-provider-env0/pull/42/commits/00a2a8b6117b316725800e470b04b4f63f690ce9